hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/RunBootstrapProcedure.scala

```diff
@@ -110,9 +111,7 @@ class RunBootstrapProcedure extends BaseProcedure with ProcedureBuilder with Log
 
     val cfg = new BootstrapExecutorUtils.Config()
     cfg.setTableName(tableName)
-    if (database.isDefined) {
-      cfg.setDatabase(database.get)
-    }
+    cfg.setDatabase(database.getOrElse(HIVE_DATABASE.defaultValue()))
     cfg.setTableType(tableType)
```

Comment:
The option `DataSourceWriteOptions.HIVE_DATABASE` is already deprecated.

Reply:
Does the spark sqlcontext contains info about the database name?

Reply:
yes, we can get the current database by `sparkSession.sessionState.catalog.getCurrentDatabase`. Do we need to change to that?

Reply:
I think so. Using the session database name makes more sense.